### DISCUSSION TITLE
how to mantain the filter settings

### DISCUSSION THREAD

**Discussion by DocS788** [ChristopherKlay/StadiaEnhanced#185] [Q&A]
Hi great community :)

how can i keep the settings of the new filter image without keep open the window ?

**Comment by ChristopherKlay** [comment]
If you mean keeping access to the filter settings while playing and having the menu open - you can't.
Once you close the menu, Stadia focuses on the game and any mouse/keyboard inputs are redirected to it.

You can only change the filter, while the menu is open, due to this.

**Reply by DocS788** [reply]
> If you mean keeping access to the filter settings while playing and having the menu open - you can't. Once you close the menu, Stadia focuses on the game and any mouse/keyboard inputs are redirected to it.
> 
> You can only change the filter, while the menu is open, due to this.

very kind mate;  so the window must be opened to take effect OK. In this case maybe an improved transparency could it possible? playing game like FIFA is a bit difficult

**Reply by ChristopherKlay** [reply]
The menu doesn't need to be open for the filter to work, it only needs to be open to change the settings.

You basically open it, change the settings to your liking and then click the filter icon again to hide it and play normally (:
